dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/net/info/BroadcastReceiverNetworkInfoProviderTest.kt

```diff
@@ -84,20 +85,20 @@ internal class BroadcastReceiverNetworkInfoProviderTest {
 
     @Test
     fun `it will do nothing if unregister is called before register`() {
-        // when
+        // When
```

Comment:
You forgot to change the methods name but I guess we could do this in another PR.

Reply:
This was a simple Search and replace, I didn't do it manually :D